tests/Unit/PointwiseFunctions/AnalyticSolutions/Elasticity/Test_BentBeam.cpp

```diff
@@ -50,10 +52,15 @@ struct BentBeamProxy : Elasticity::Solutions::BentBeam {
     return Elasticity::Solutions::BentBeam::variables(x, field_tags{});
   }
 
+  // NOLINTNEXTLINE(readability-convert-member-functions-to-static)
   tuples::tagged_tuple_from_typelist<source_tags> source_variables(
       const tnsr::I<DataVector, 2>& x) const noexcept {
     return Elasticity::Solutions::BentBeam::variables(x, source_tags{});
   }
+
+  double potential_energy() const noexcept {
+    return Elasticity::Solutions::BentBeam::potential_energy();
+  }
```

Comment:
This addition should not be necessary, the `potential_energy` function should be inherited. Either way, you can call the function on the `BentBeam` itself instead of the proxy (the proxy is only needed for the random python tests)